virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc

```diff
@@ -1,23 +1,21 @@
 :_content-type: ASSEMBLY
-[id="virt-configuring-local-storage-for-vms"]
+[id=""]
```

Comment:
The assembly needs to have a unique ID per our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata)

Reply:
Thanks, @sjhala-ccs. How do I find out what the missing id should be?

Reply:
The assembly ID is usually the file name. So, in this case:
````
[id="virt-configuring-local-storage-for-vms"]
````
That is what it was before as you can see by the red highlighting above.